core/templates/pages/story-editor-page/navbar/story-editor-navbar.component.ts

```diff
@@ -96,40 +96,44 @@ export class StoryEditorNavbarComponent implements OnInit {
 
   discardChanges(): void {
     this.undoRedoService.clearChanges();
-    this.storyEditorStateService.loadStory(this.story.getId());
+    if (this.story) {
```

Comment:
What else are we expecting here `this.story` to be ? `null` / `undefined` ?

Reply:
The story remains undefined for some period of time till we fetch the story from the backend. So to prevent this function from getting called at that time we are using this check.

Reply:
I don't know why but this is causing the lighthouse test to fail.

Reply:
This check is only needed because for a short period of time story remains undefined (while fetching backend values).